internal/pkg/template/templates/workloads/services/rd-web/manifest.yml

```
@@ -19,24 +19,40 @@ image:
 {{- end}}
   # Port exposed through your container to route traffic to it.
   port: {{.ImageConfig.Port}}
-
+{{if .Private.IsBasic}}
```

Was this on purpose?
```suggestion
{{- if .Private.IsBasic}}
```

---

yeah 🥺 otherwise it looks like
```
  port: 8080
http:
  private: true
```
and i wanted to keep the "section space" between `port`/`http`